owncloudApp/src/main/java/com/owncloud/android/ui/fragment/UploadListFragment.java

```diff
@@ -104,6 +106,31 @@ public boolean onChildClick(ExpandableListView parent, View v, int groupPosition
 
     }
 
+    public enum OptionsInUploadList {
+        CLEAR_FAILED, CLEAR_SUCCESSFUL, RETRY_FAILED;
+    }
+
+    @Override
+    public void onClick(OptionsInUploadList option) {
```

Comment:
This is not toolbar related, isn't it?

Reply:
Well, it is related because we need to remove the three-dot menu from the uploads fragment to set the avatar. So, we have moved those options to the fragment.

Reply:
We don't have a viewmodel here, right? To move the `TransferRequester` and `UploadsStorageManager` dependencies there and call viewmodel methods here.

Reply:
Not yet, unfortunately.